cram_common/cram_common_failures/src/failure-handling-strategies.lisp

```diff
@@ -91,7 +95,7 @@ after each iteration of the retry."
        (cpl:fail ,rethrow-failure))))
 
 (defun next-different-location-solution (designator
-                                         &optional (distance-threshold 0.05))
+                                         &optional (distance-threshold *default-distance-threshold*))
```

Comment:
The linebreak would probably be nice here, although I did not count the characters, so I wouldn't know.